austation/code/modules/admin/verbs/kill_rads.dm

```diff
@@ -0,0 +1,24 @@
+var/static/list/notradkillable = typesof(/obj/item/twohanded/required/fuel_rod) + typesof(/obj/machinery/atmospherics/components/trinary/nuclear_reactor)
```

Comment:
There's only ever going to be one instance of this, make a global list that objects can add themselves to on initialization.
Ideally, add a flag like KILL_RAD_IMMUNE and make the below proc check for it. Hardcoded lists for stuff like this is more or less a bad idea

Reply:
I just forgot smh 😔